src/codegen/compile.ml

```diff
@@ -5551,7 +5551,7 @@ module MakeSerialization (Strm : Stream) = struct
       E.else_trap_with env "data buffer not filled" ^^
 
       get_refs_size ^^
-      G.i (Test (Wasm.Values.I32 I32Op.Eqz)) ^^
+      compile_unboxed_const 0l ^^
```

Comment:
`compile_eq_const`, not `compile_unboxed_const`

Reply:
Yeah, silly me. Copied from the wrong place. Corrected.